### PR TITLE
[StaticRuntime] Use NNC's call_raw API to reduce call overheads.

### PR DESCRIPTION
**Pull request by ZolotukhinM**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#57329 [StaticRuntime] Use NNC's call_raw API to reduce call overheads.**
* #57328 [TensorExpr] Add `TensorExprKernel::runFast` method.
* #55113 [TensorExpr] Add `CodeGen::call_raw` method.

Differential Revision: [D28110358](https://our.internmc.facebook.com/intern/diff/D28110358)

### CODE REVIEW

File: torch/csrc/jit/runtime/static/ops.cpp [hunk 1]
```diff
@@ -379,6 +379,10 @@ struct TEWrapper {
     cg->call(args);
   }
```
Comment (bertmaher) [inline torch/csrc/jit/runtime/static/ops.cpp:380]: Can we rip out the `operator()` overload if we're not going to use it anymore?

**Review by bertmaher — APPROVED** [review]
wicked!

File: torch/csrc/jit/runtime/static/ops.cpp (hunk 1 above)
Comment (ZolotukhinM) [reply torch/csrc/jit/runtime/static/ops.cpp:380]: We probably can

Comment (ZolotukhinM) [reply torch/csrc/jit/runtime/static/ops.cpp]: Done!

**Comment by mruberry** [comment]
Unlanding this stack, as it appears to be breaking pytorch_linux_xenial_py3_clang5_asan_test2. Failure snippet:

```
Apr 30 20:18:38 [ RUN      ] Kernel.RunFast
Apr 30 20:18:38 unknown file: Failure
Apr 30 20:18:38 C++ exception with description "SimpleIREvaluator::call_raw is not implemented yet" thrown in the test body.
Apr 30 20:18:38 [  FAILED  ] Kernel.RunFast (7 ms)
```

The failure appears in the PR CI

**Pull request closed without merging**